### PR TITLE
[CORE-1816] Workorder draft location

### PR DESCRIPTION
**Pull request by JordyDeGroeve**
Changed the folder structure to reflect the grouping better.

+ added the location to the .draft endpoint for workorders.

## Manual Check
run 
```
make all && make preview && open apiary.html
```
Check if the `location` is added to **workOrders.draft** 
and that the `time-tracking` group is still loaded correctly.

### CODE REVIEW

**Review by tiagobastos — APPROVED** [review]
✅ The location (with Address object) is added to the `workorders.draft` endpoint and the time-tracking group is still loaded.

**Review by WouterSioen — APPROVED** [review]

File: src/09-workorders/work-orders.apib [hunk 1]
```diff
@@ -21,6 +21,7 @@ Add a draft work order.
            + (object)
                + product_id: `e2314517-3cab-4aa9-8471-450e73449040` (string, required)
                + quantity: 3 (number, required)
+       + location (Address)
```
Comment (dietrichm) [inline src/09-workorders/work-orders.apib]: B:
```suggestion
       + location (Address, optional)
```

**Review by dietrichm — CHANGES_REQUESTED** [review]

**Review by dietrichm — APPROVED** [review]

File: apiary.apib [hunk 2]
```diff
@@ -3306,6 +3306,7 @@ Add a draft work order.
            + (object)
                + product_id: `e2314517-3cab-4aa9-8471-450e73449040` (string, required)
                + quantity: 3 (number, required)
+       + location (Address)
```
Comment (WouterSioen) [inline apiary.apib]: B: this still needs to be build
```suggestion
       + location (Address, optional)
```

**Review by WouterSioen — CHANGES_REQUESTED** [review]

File: apiary.apib (hunk 2 above)
Comment (JordyDeGroeve) [reply apiary.apib]: 🤦‍♂️

**Review by dietrichm — DISMISSED** [review]
Voilá 💁‍♂️

**Review by WouterSioen — DISMISSED** [review]

**Review by jenssegers — DISMISSED** [review]

**Comment by dietrichm** [comment]
🚨 Conflicts

**Comment by WouterSioen** [comment]
Rebased to fix the conflicts, to be able to work further on this.

**Comment by WouterSioen** [comment]
✅ looks good

**Review by tiagobastos — APPROVED** [review]

**Review by WouterSioen — APPROVED** [review]
Reviews got dropped by changing the base branch. Still approving

**Review by dietrichm — APPROVED** [review]

**Pull request merged**